plugins/scaffolder-backend/src/scaffolder/tasks/logger.ts

```diff
@@ -21,6 +21,7 @@ import { JsonObject } from '@backstage/types';
 import { Format, TransformableInfo } from 'logform';
 import Transport, { TransportStreamOptions } from 'winston-transport';
 import { Logger, format, createLogger, transports } from 'winston';
+import { MESSAGE } from 'triple-beam';
```

Comment:
Wanna note that this is implemented with `Symbol.for(...)`, so duplication is not an issue.